dummy/src/main/resources/README.md

```diff
@@ -0,0 +1,11 @@
+# README
+
+You can execute the driver with three modes, and you should update the mode in the properties file. 
+- CREATE_VALIDATION
+- VALIDATE_DATABASE
+- EXECUTE_BENCHMARK
+
+
+```
+java -cp xxx.jar org.ldbcouncil.finbench.driver.driver.Driver -P example.properties
```

Comment:
commit the example data to this repo.

Reply:
And remove the data, configs from the driver.

Reply:
During development, data and configs are needed on the driver.  Is it better to put the data and configs on the driver first？

Reply:
Discussed offline